javascripts/background_scripts/reading_process.js

```diff
@@ -140,56 +38,26 @@ var readingProcess = {
     var pattern = /privlyInjectableApplication\=/i;
     url = url.replace(pattern, "privlyApp=");
 
-    if( url.indexOf("privlyApp=Message") > 0 ) {
+    if (url.indexOf("privlyApp=Message") > 0) {
       path = "privly-applications/Message/show.html?privlyOriginalURL=";
-    } else if( url.indexOf("privlyApp=ZeroBin") > 0) {
+    } else if (url.indexOf("privlyApp=ZeroBin") > 0) {
       path = "privly-applications/Message/show.html?privlyOriginalURL="; // Deprecated
-    } else if( url.indexOf("privlyApp=PlainPost") > 0) {
+    } else if (url.indexOf("privlyApp=PlainPost") > 0) {
       path = "privly-applications/PlainPost/show.html?privlyOriginalURL=";
-    } else if( url.indexOf("https://priv.ly") === 0 ) {
+    } else if (url.indexOf("https://priv.ly") === 0) {
       path = "privly-applications/PlainPost/show.html?privlyOriginalURL="; // Deprecated
     } else {
-      sendResponse({}); // Don't inject unknown apps
```

Comment:
Oops. Sorry seems that this inconformity is introduced in migerating from Chrome message to Privly message. https://github.com/SummerWish/privly-chrome/commit/43734a3218919070537ca776f412d73fb2670a31#diff-8f9a02de8a0c418341bf437eced8daabR50 I will fix it :-)